File: 20495_corejava/Day2Assignments/Car.java
package com.sonata;

public class Car {
	
	int speed;
	double regularPrice;
	String color;
	static {
		System.out.println("Show the static block");
	}
	
	Car(){}
	Car(int speed,double regularPrice,String color){
		this.speed=speed;
		this.regularPrice=regularPrice;
		this.color=color;
	}
	public void getSalePrice() {
		
		System.out.println(regularPrice);
	}
	public void display()
	{
		System.out.println(speed);
		System.out.println(regularPrice);
		System.out.println(color);
	}

}

File: TypeScriptAssignment/Car.js
"use strict";
exports.__esModule = true;
exports.Car = void 0;
var Car = /** @class */ (function () {
    function Car(speed, regularPrice, color) {
        this.speed = speed;
        this.regularPrice = regularPrice;
        this.color = color;
    }
    Car.prototype.getSalePrice = function (gst) {
        this.salePrice = this.regularPrice + (gst / 100 * this.regularPrice);
    };
    Car.prototype.display = function () {
        console.log(this.speed);
        console.log(this.regularPrice);
        console.log(this.color);
        console.log(this.salePrice);
    };
    return Car;
}());
exports.Car = Car;
